src/reader.rs

```diff
@@ -38,7 +38,7 @@ use error::{from_io_err, from_parse_err};
 /// use config::reader::from_stream;
 /// use config::error::ConfigErrorKind;
 ///
-/// let sample_conf = "windows=\n";
+/// let sample_conf = "windows=NO SPACE\n";
```

Comment:
Why `windows=NO SPACE`? A configuration containing `windows=` should still fail to parse.